deploy/charts/kubevirtbmc/templates/serviceaccount.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "chart.serviceAccountName" . }}-manager
+  name: {{ include "chart.name" . }}-controller-manager
```

Comment:
Let's keep the `{{ include "chart.serviceAccountName" . }}` template as we do expose the configurable in the `values.yaml`.